Containerfile.zero-trust-workload-identity-manager.bundle

```
@@ -40,15 +40,15 @@ LABEL com.redhat.component="zero-trust-workload-identity-manager-bundle-containe
       io.openshift.build.source-location="${SOURCE_URL}" \
       io.openshift.build.commit.url="${SOURCE_URL}/commit/${COMMIT_SHA}" \
       io.openshift.maintainer.product="OpenShift Container Platform" \
-      io.openshift.tags="openshift,secrets,external-secrets,zero-trust-workload-identity-manager" \
+      io.openshift.tags="openshift,security,zero-trust-workload-identity-manager" \
       io.k8s.display-name="openshift-zero-trust-workload-identity-manager-bundle" \
       io.k8s.description="zero-trust-workload-identity-manager-bundle-container" \
       operators.operatorframework.io.bundle.mediatype.v1="registry+v1" \
       operators.operatorframework.io.bundle.manifests.v1=manifests/ \
       operators.operatorframework.io.bundle.metadata.v1=metadata/ \
       operators.operatorframework.io.bundle.package.v1="openshift-zero-trust-workload-identity-manager" \
-      operators.operatorframework.io.bundle.channel.default.v1="stable-v1" \
-      operators.operatorframework.io.bundle.channels.v1="stable-v1,stable-v0.14" \
+      operators.operatorframework.io.bundle.channel.default.v1="tech-preview" \
+      operators.operatorframework.io.bundle.channels.v1="tech-preview" \
```

Should this be like below?
```suggestion
      operators.operatorframework.io.bundle.channels.v0="techpreview-v0.1" \
```
@swghosh

---

It is also fine to have just `tech-preview` as the channel since we plan to ship a few versions to TP before hitting GA. Also, the naming highly depends on the lifecycle you want to keep in place for your operator - if there is a requirement for us to allow users to stay on tech preview v0.1.z versions then the channel tech-preview-v0.1 is required else not.